### ISSUE TITLE
Where to move old files in the source?

### ISSUE THREAD

**Issue by larsenwork** [fontforge/fontforge#2328]
I'm in the process of updating app icons and pixmaps and was wondering if it was smarter to make a top level archive folder so that graphics and code that is no longer used is in one place instead of being scattered throughout the source tree?

**Comment by larsenwork** [comment]
does @davelab6 @frank-trampe or someone else have an opinion on this?

**Comment by frank-trampe** [comment]
I don't think that it's worth the trouble. I prefer to have any legacy stuff that's useful close to its original location; otherwise, one might not think to look for it.

**Comment by davelab6** [comment]
The old position is bad. The proposed position is good. Change aversion is
temporary.
